test/kubernetes/e2e/test.go

```diff
@@ -133,8 +139,12 @@ type TestInstallation struct {
 	Actions *actions.Provider
 
 	// Assertions is the entity that creates assertions that can be executed by the Operator
+	// DEPRECATED: Use AssertionsT instead (which is scoped to a specific test and not the root suite)
 	Assertions *assertions.Provider
 
+	// AssertionsT creates an assertion provide that is scoped to the provided test
```

Comment:
```suggestion
	// AssertionsT creates an assertion provider that is scoped to the provided test
```

nit: noticed a typo, but feel free to ignore this if you want